roles/openstack-setup/tasks/flavors.yml

```diff
@@ -21,6 +26,7 @@
       password: "{{ secrets.admin_password  }}"
   with_items: '{{ nova.flavors | default([]) }}'
   run_once: true
+  when: flavorcount.stdout == ""
   # FIXME flavors are always added, even if it's just an Ironic cloud
```

Comment:
nope, so let's leave the logic in your PR as we discussed and add a flag to allow force recreate if needed.